sdk/keyvault/keyvault-keys/src/core/models/index.ts

```diff
@@ -8,2967 +8,2251 @@
  * regenerated.
  */
 
-import { BaseResource, CloudError } from "@azure/ms-rest-azure-js";
-import * as msRest from "@azure/ms-rest-js";
+import { BaseResource, CloudError } from "@azure/core-lro";
+import * as msRest from "@azure/core-http";
 
 export { BaseResource, CloudError };
 
 /**
- * @interface
- * An interface representing Attributes.
  * The object attributes managed by the KeyVault service.
- *
  */
 export interface Attributes {
   /**
-   * @member {boolean} [enabled] Determines whether the object is enabled.
```

Comment:
I spoke with @kpajdzik and he confirmed that the removal of these typing strings in the generated JSDoc comments is expected due to some recent changes in `autorest.typescript`.